apps/vesting/src/components/VestingAccountStats.tsx

```diff
@@ -139,6 +144,11 @@ function LockupTimelineListItem({
   const transactionTimestamp = useMemo(() => {
     return formatDate(date);
   }, [date]);
+  const [parsedAmount, formattedMount] = useMemo(() => {
```

Comment:
We have new function 'formatEthDecimal' 

It formats all eth-based numbers

Reply:
noted. we change this in other pr.